### PR TITLE
Setting up repo

### PR DESCRIPTION
**Pull request by mauro-fanton**

### CODE REVIEW

**Review by mpbostock — APPROVED** [review]

**Review by mpbostock — CHANGES_REQUESTED** [review]
There are some Typos "Sustem" should be "System"

**Comment by mauro-fanton** [comment]
Yes, this is a wip. we will need to go trough it and changes all this reference. I have put some note on the ticket.

**Comment by ChrisCookOC** [comment]
Generally we have used excise-movement-control-system-tis in a few places instead of excise-movement-control-system-api-tis. Is this intentional / does it matter?

File: source/documentation/partials/_BR003.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+ <tr>
+  <td colspan="3">
+   It is obligatory that the reference of tax warehouse of the place of dispatch that is included in the draft e-AD exists in the using tax warehouse that is included in the trader 
+
+authorisation, that is registered in seed for the consignor of the movement, if the origin type code of the draft e-AD is Origin-tax warehouse.
+  </td>
+ </tr>
+ <tr>
+  <td colspan="3">
+   - if the type of origin is tax warehouse,ooptionally, validity of the link (manages or uses) between consignor and place of dispatch
```
Comment (ChrisCookOC) [inline source/documentation/partials/_BR003.md:23]: "ooptionally" typo

File: source/documentation/partials/_BR005.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  <td colspan="3">
+   It is obligatory that the reference number of the office responsible for release for free circulation that is included in the draft e-AD exists in the customs office list, if the origin type code of the draft e-AD is Origin-import.
+  </td>
+ </tr>
+ <tr>
+  <td colspan="3">
+   - if the type of origin is import,
+
+
+ovalidity of the code of the Customs office of import
```
Comment (ChrisCookOC) [inline source/documentation/partials/_BR005.md:24]: "ovalidity" typo

File: source/documentation/partials/_BR009.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  <td colspan="3">
+   It is obligatory that the transport mode code of the transport mode that is included in the draft e-AD is Sea transport or Inland waterway transport, if the destination type code of the draft e-AD is Unknown destination.
+  </td>
+ </tr>
+ <tr>
+  <td colspan="3">
+   - if destination data are left empty (Article 22 of Directive 2020/262),
+
+
+othe transport mode is either sea or inland waterway
```
Comment (ChrisCookOC) [inline source/documentation/partials/_BR009.md:24]: "othe" typo

File: source/documentation/partials/_BR010.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  <td colspan="3">
+   It is obligatory that the operator role code of at least one operator role that is included in the trader authorisation, that is registered in seed for the consignor of the movement, is Allowed to leave empty the destination fields according to article 22 of the directive 2020/262, if the destination type code of the draft e-AD is Unknown destination.
+  </td>
+ </tr>
+ <tr>
+  <td colspan="3">
+   - if destination data are left empty (Article 22 of Directive 2020/262,
+
+
+othe consignor is allowed to leave these fields empty
```
Comment (ChrisCookOC) [inline source/documentation/partials/_BR010.md:24]: "othe" typo

File: source/documentation/partials/_BR011.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+  <td colspan="3">
+   It is obligatory that the excise product code of each body e-AD that is included in the draft message exists in the trader authorisation that is registered in seed for the consignee that is included in the draft message, if the destination type code of the draft message is Destination-registered consignee or Destination-direct delivery.
+  </td>
+ </tr>
+ <tr>
+  <td colspan="3">
+   - if the consignee is a registered consignee:
+
+
+onature of goods compared to the categories and/or products allowed for the consignee;
```
Comment (ChrisCookOC) [inline source/documentation/partials/_BR011.md:24]: "onature" typo

File: source/documentation/partials/_BR011.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+   - if the consignee is a registered consignee:
+
+
+onature of goods compared to the categories and/or products allowed for the consignee; 
+
+
+- if the consignment is for direct delivery:
+
+
+onature of goods compared to the categories and/or products allowed for the consignee.
```
Comment (ChrisCookOC) [inline source/documentation/partials/_BR011.md:30]: "onature" typo

File: source/documentation/partials/_BR012.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+  <td colspan="3">
+   It is obligatory that the trader id of the delivery place that is included in the draft message is equal to the reference of tax warehouse of the using tax warehouse that is included in the trader authorisation, that is registered in seed for the consignee of the draft message, if the destination type code of the draft message is Destination-tax warehouse.
+  </td>
+ </tr>
+ <tr>
+  <td colspan="3">
+   - if the (new) place of delivery is a tax warehouse,
+
+
+ovalidity of the link (manages or uses) between (new) consignee and (new) place of delivery
```
Comment (ChrisCookOC) [inline source/documentation/partials/_BR012.md:24]: "ovalidity" typo

File: source/documentation/partials/_BR017.md [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+  <td colspan="3">
+   It is obligatory that the reference number of the delivery place customs office that is included in the draft message exists in the customs office list with export role, if the destination type code of the draft message is Destination-export.
+  </td>
+ </tr>
+ <tr>
+  <td colspan="3">
+   - if the consignment is for export,
+
+
+o validity of the Customs office of export
```
Comment (ChrisCookOC) [inline source/documentation/partials/_BR017.md:24]: random o again

File: source/documentation/partials/_BR014.md [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+  <td colspan="3">
+   It is obligatory that the operator role code of at least one operator role that is included in the trader authorisation, that is registered in seed for the consignee of the draft message, is Allowed to practice direct delivery, if the destination type code of the draft message is Destination-direct delivery.
+  </td>
+ </tr>
+ <tr>
+  <td colspan="3">
+   - if the consignment is (now) for direct delivery,
+
+
+o the (new) consignee is allowed to practise direct delivery
```
Comment (ChrisCookOC) [inline source/documentation/partials/_BR014.md:26]: random o

File: source/documentation/partials/_BR027.md [hunk 10]
```diff
@@ -0,0 +1,37 @@
...
+ 
+It is obligatory that the reference number of the destination office that is included in the draft alert or rejection of an e-SAD is equal to the reference number of issuance office of the temporary authorisation that is registered in seed for the temporary certified consignee that is included in the draft alert or rejection of an e-SAD, if the destination type code of the draft alert or rejection of an e-SAD is Destination-temporary certified consignee.
+  </td>
+ </tr>
+ <tr>
+  <td colspan="3">
+   - if the consignee is a temporary registered consignee,
+
+
+othe code of the Excise office is mentioned in the temporary authorisation
```
Comment (ChrisCookOC) [inline source/documentation/partials/_BR027.md:28]: "othe"

File: source/documentation/partials/_BR031.md [hunk 11]
```diff
@@ -0,0 +1,33 @@
...
+  <td colspan="3">
+   It is obligatory that the excise product code of each body e-AD that is included in each split details e-AD contained in the draft splitting operation corresponds to an excise products category code in the splitting excise products category parameters of the reference data in seed.
+  </td>
+ </tr>
+ <tr>
+  <td colspan="3">
+   - the following must be globally verified: 
+
+
+onature of goods compared to the categories allowed for splitting (energy products);
```
Comment (ChrisCookOC) [inline source/documentation/partials/_BR031.md:24]: "onature"

File: source/documentation/partials/_BR031.md [hunk 12]
```diff
@@ -0,0 +1,33 @@
...
+   - the following must be globally verified: 
+
+
+onature of goods compared to the categories allowed for splitting (energy products); 
+
+
+- the following must be verified separately for each part resulting from the splitting:
+
+
+oif destination data are left empty (Article 22 - at most once for the submission), the goods are subject to splitting according to the applicable common system parameter
```
Comment (ChrisCookOC) [inline source/documentation/partials/_BR031.md:30]: "oif"

File: source/documentation/partials/_BR032.md [hunk 13]
```diff
@@ -0,0 +1,27 @@
...
+  <td colspan="3">
+   It is obligatory that the excise product code of each body e-AD that is included in each split details e-AD contained in the draft splitting operation exists in at least one body e-AD that is included in the e-AD.
+  </td>
+ </tr>
+ <tr>
+  <td colspan="3">
+   - compared to the former e-AD,
+
+
+ono product has been added in any of the new e-Ads
```
Comment (ChrisCookOC) [inline source/documentation/partials/_BR032.md:24]: "ono"

File: source/documentation/partials/_BR034.md [hunk 14]
```diff
@@ -0,0 +1,27 @@
...
+  <td colspan="3">
+   It is permitted that the destination type code of at most one split details e-AD that is included in the draft splitting operation is Unknown destination, if the destination type code of the e-AD is Unknown destination.
+  </td>
+ </tr>
+ <tr>
+  <td colspan="3">
+   - if destination data are left empty (Article 22 - at most once for the submission),
+
+
+othe former e-AD had an empty destination
```
Comment (ChrisCookOC) [inline source/documentation/partials/_BR034.md:24]: "othe"

File: source/documentation/partials/_BR048.md [hunk 15]
```diff
@@ -0,0 +1,27 @@
...
+  <td colspan="3">
+   It is obligatory that the excise product code of each body e-SAD that is included in the draft message exists in the trader authorisation that is registered in seed for the consignee that is included in the draft message, if the destination type code of the draft message is Destination-certified consignee.
+  </td>
+ </tr>
+ <tr>
+  <td colspan="3">
+   - if the consignee is a certified consignee:
+
+
+onature of goods is the same compared to the categories and/or products allowed for the consignee;
```
Comment (ChrisCookOC) [inline source/documentation/partials/_BR048.md:24]: "onature"

**Comment by mpbostock** [comment]
I wonder if all these "typos" that Chris is finding is a flaw of the Python extraction from the DDNEA?? I know that CTC had to get the whole team to jump on and go through the extracted DDNTA and make sure the extracted pages made sense and were human readable. We will have to clean this up generally I think but is a good starting point at least. We also need to make sure that we don't have any rules/constraints etc. left over from the CTC copy as that would definitely be very confusing! 😂

**Comment by ChrisCookOC** [comment]
@mpbostock  yeah it looks like it only happens on certain types of lines so its a conversion thing I'm sure!

**Comment by mauro-fanton** [comment]
> Generally we have used excise-movement-control-system-tis in a few places instead of excise-movement-control-system-api-tis. Is this intentional / does it matter?

yes I agree. We might can rename it.

**Comment by mauro-fanton** [comment]
yes, probably there is a problem with the python script. Yes we would need be careful with this as it has been a copy and paste job  as we did not know what info we needed. I think it will require every body efors, and specially James to see if we have the right info.

**Comment by crystalmark** [comment]
Slopey shoulders...  all typos have been auto-imported from the DDNEA :-)

**Review by mpbostock — APPROVED** [review]

**Pull request merged**